docs/docs/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Get Started
+layout: docs
+---
+
+Use the Tableau Server Client (TSC) library to increase your productivity as you interact with the Tableau Server REST API. With
+the TSC library you can do almost everything that you can do with the REST API, including:
+
+* Publish workbooks and data sources.
+* Create users and groups.
+* Query projects, sites, and more.
+
+This section describes how to:
+
+* TOC
+{:toc}
+
+## Confirm prerequisites
+
+Before you install TSC, confirm that you have the following dependencies installed:
+
+* Python. You can use TSC with Python 2.7.9 or later and with Python 3.3 or later. These versions include pip, which is
+  the recommended means of installing TSC.
+* Git. Optional, but recommended to download the samples or install from the source code.
+
+
+## Install TSC
+
+You can install TSC with pip or from the source code.
+
+### Install with pip (recommended)
+
+Run the following command to install the latest stable version of TSC:
+
+```
+pip install tableauserverclient
+```
+
+### Install from the source code
+
+Run the following command to install directly from the TSC repository:
+
+```
+pip install git+https://github.com/tableau/server-client-python.git
+```
+
+Alternatively, you can install from the development branch for a preview of upcoming features. Run the following command
+to install from the development branch:
+
+```
+pip install git+https://github.com/tableau/server-client-python.git@development
+```
+
+Note that the version from the development branch should not be used for production code. The methods and endpoints in the
+development version are subject to change at any time before the next stable release.
```

Comment:
We should call out that until we hit 1.0, there may be breaking changes in the API, but afterwards, we will be following a semvar numbering scheme.